templates/ts/custom-copilot-rag-azure-ai-search/README.md.tpl

```diff
@@ -0,0 +1,90 @@
+# Overview of the AI Search Bot template
+
+This app template is built on top of [Teams AI library](https://aka.ms/teams-ai-library).
+It showcases how to build an basic RAG bot in Teams capable of chatting with users but with context provided by Azure AI Search data source.
+
+- [Overview of the AI Search Bot template](#overview-of-the-ai-search-bot-template)
+  - [Get started with the AI Search Bot template](#get-started-with-the-ai-search-bot-template)
+  - [What's included in the template](#whats-included-in-the-template)
+  - [Extend the AI Search Bot template with more AI capabilities](#extend-the-ai-search-bot-template-with-more-ai-capabilities)
+  - [Additional information and references](#additional-information-and-references)
+
+## Get started with the AI Search Bot template
+
+> **Prerequisites**
+>
+> To run the AI Search bot template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-toolkit-cli)
+{{#useOpenAI}}
+> - An account with [OpenAI](https://platform.openai.com/) and [Azure AI Search](https://azure.microsoft.com/en-us/products/ai-services/ai-search).
+{{/useOpenAI}}
+{{#useAzureOpenAI}}
+> - Prepare your own [Azure OpenAI](https://aka.ms/oai/access) resource and [Azure AI Search](https://azure.microsoft.com/en-us/products/ai-services/ai-search).
+{{/useAzureOpenAI}}
+
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+{{#useOpenAI}}
+1. In file *env/.env.testtool.user*, fill in your OpenAI key `SECRET_OPENAI_API_KEY=<your-key>`. And fill in your Azure AI search key `SECRET_AZURE_SEARCH_KEY=<your--ai-search-key>` and endpoint `AZURE_SEARCH_ENDPOINT=<your-ai-search-endpoint>`.
```

Comment:
minor:  As there is no <your-key> placeholder in .env.*.user, do we need to use this placeholder here?  @MuyangAmigo What's your opinion?